### DISCUSSION TITLE
Is this project still maintained?

### DISCUSSION THREAD

**Discussion by Mexator** [requarks/wiki#7324] [General]
I am looking at the list of PRs and there seems to be not much activity. Why is is it like that? 

Are PRs not welcome for the time being, or maintainer(s) are too busy doing v3?

What do I do to make my PR merged into v2?

Sorry if I may seem rude, I am genuinely curious about that

**Comment by mvtab** [comment]
You should check out v3. Game changer.

**Reply by Mexator** [reply]
The blog has no updates for around a year. The docs are scarce. It explicitly says "Don't use v3 in production". So I'm thinking it's a bit early to use it.

Am I wrong? Could you elaborate why v3 is game changer?

**Comment by mvtab** [comment]
I don't mean to use it in production, just try it out in a test environment and you will see what I mean. Everything is changed, I think they even have a completely new code base.

It's understandable that they don't invest much in v2 anymore.
